example.env

```diff
@@ -18,3 +18,6 @@ WEBHOOK_LIMIT=
 ROLLBAR_ACCESS_TOKEN=
 ENVIRONMENT=
 WEBHOOK_TOKEN_SECRET_KEY=
+
+SECRET_KEY_2FA= // this has to be base32 to work with Google Authenticator
```

Comment:
hmm, I wouldn't add comments to `example.env`. If anywhere, I'd put it in README.md